src/openrct2/management/NewsItem.cpp

```diff
@@ -70,7 +70,8 @@ NewsItem * news_item_get(sint32 index)
 
 bool news_item_is_empty(sint32 index)
 {
-    return news_item_get(index)->Type == NEWS_ITEM_NULL;
+    NewsItem * news = news_item_get(index);
+    return news && news->Type == NEWS_ITEM_NULL;
```

Comment:
So `news` should not be `NULL`, but `news->Type` should be `NEWS_ITEM_NULL`? Is that the intention?

Reply:
`return news != nullptr && news->Type == NEWS_ITEM_NULL;`